src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanCreateNewConfig()
                     f.WriteObject(
                         It.Is<AzureHDInsightConfig>(
                             c =>
-                                c.ClusterType == HDInsightClusterType.Hadoop &&
+                                c.ClusterType == "Hadoop" &&
```

Comment:
Fixed by adding constants. Insensitive comparison is not needed as the comparison only happens in test code.